exodus_lambda/functions/origin_request.py

```diff
@@ -103,6 +149,40 @@ def resolve_aliases(self, uri):
 
         return uri
 
+    def handle_cookie_request(self, event):
+        signer = Signer(self.cookie_key, self.conf.get("key_id"))
+        expiration = timedelta(minutes=int(self.conf.get("cookie_ttl", "720")))
+        domain = event["Records"][0]["cf"]["config"]["distributionDomainName"]
+        uri = event["Records"][0]["cf"]["request"]["uri"]
+
+        cookies_content = signer.cookies_for_policy(
+            append="; Secure; Path=/content/; Max-Age=%s"
+            % int(expiration.total_seconds()),
+            resource="https://%s/content/*" % domain,
+            date_less_than=datetime.utcnow() + expiration,
```

Comment:
Would you mind calling utcnow() just once in here, so that the cookies always come in a pair?

If it's called more than once, time can pass between the two which means the origin and content cookies can expire at slightly different times.  Not a big deal, it just makes it harder to match them up as "these were generated together" while debugging.